Performing C SOURCE FILE Test HAVE_TM_GMTOFF failed

Jerry version:

    Checked revision: 85e0509
OS:

    ubuntu-14.04.5-desktop-amd64

Build command: 
    
    python tools/build.py

There are two errors show in the file `/jerryscript/build/CMakeFiles/CMakeError.log` as following:

    Performing C SOURCE FILE Test HAVE_TM_GMTOFF failed with the following output:

    Change Dir: /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp

    Run Build Command:/usr/bin/make "cmTryCompileExec1239510574/fast"
    /usr/bin/make -f CMakeFiles/cmTryCompileExec1239510574.dir/build.make CMakeFiles/cmTryCompileExec1239510574.dir/build
    make[1]: Entering directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
    /usr/bin/cmake -E cmake_progress_report /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CMakeFiles 1
    Building C object CMakeFiles/cmTryCompileExec1239510574.dir/src.c.o
    /usr/bin/cc  -D_BSD_SOURCE -D_DEFAULT_SOURCE -flto -fno-fat-lto-objects -std=c99 -pedantic -fno-builtin -fno-stack-protector -Wall -Werror=all -Wextra -Werror=extra -Wformat-nonliteral -Werror=format-nonliteral -Winit-self -Werror=init-self -Wconversion -Werror=conversion -Wsign-conversion -Werror=sign-conversion -Wformat-security -Werror=format-security -Wmissing-declarations -Werror=missing-declarations -Wshadow -Werror=shadow -Wstrict-prototypes -Werror=strict-prototypes -Wundef -Werror=undef -Wold-style-definition -Werror=old-style-definition -Wno-stack-protector -Wno-attributes -Werror -Wlogical-op -Werror=logical-op -DHAVE_TM_GMTOFF -Wno-error=strict-prototypes -Wno-error=old-style-definition   -o CMakeFiles/cmTryCompileExec1239510574.dir/src.c.o   -c /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/src.c
    /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/src.c:4:5: warning: function declaration isn’t a prototype [-Wstrict-prototypes]
     int main()
     ^
    /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/src.c: In function ‘main’:
    /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/src.c:4:5: warning: old-style function definition [-Wold-style-definition]
    /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/src.c:7:4: error: statement with no effect [-Werror=unused-value]
    tmp->tm_gmtoff;
    ^
    cc1: all warnings being treated as errors
    make[1]: *** [CMakeFiles/cmTryCompileExec1239510574.dir/src.c.o] Error 1
    make[1]: Leaving directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
    make: *** [cmTryCompileExec1239510574/fast] Error 2
    
    Source file was:
    
    #include <time.h>
    
    int main()
    {
       struct tm* tmp;
       tmp->tm_gmtoff;
      return 0;
    }

Maybe for this reason, i have same problem as [issue](https://github.com/pando-project/jerryscript/issues/2193). Although i tried the solution, it still failed.

@galpeter 

```
nisl@nisl-H81M-DS2:~$ cmake --version
cmake version 2.8.12.2
```
There are only two log files, the other log file is `jerryscript/CMakeFiles/CMakeOutput.log`, and the content is:  
```
The system is: Linux - 4.4.0-142-generic - x86_64
Compiling the C compiler identification source file "CMakeCCompilerId.c" succeeded.
Compiler: /usr/bin/cc 
Build flags: 
Id flags: 

The output was:
0


Compilation of the C compiler identification source "CMakeCCompilerId.c" produced "a.out"

The C compiler identification is GNU, found in "/home/nisl/yhy/jerryscript/build/CMakeFiles/2.8.12.2/CompilerIdC/a.out"

Determining if the C compiler works passed with the following output:
Change Dir: /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp

Run Build Command:/usr/bin/make "cmTryCompileExec3662415559/fast"
/usr/bin/make -f CMakeFiles/cmTryCompileExec3662415559.dir/build.make CMakeFiles/cmTryCompileExec3662415559.dir/build
make[1]: Entering directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
/usr/bin/cmake -E cmake_progress_report /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building C object CMakeFiles/cmTryCompileExec3662415559.dir/testCCompiler.c.o
/usr/bin/cc    -o CMakeFiles/cmTryCompileExec3662415559.dir/testCCompiler.c.o   -c /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/testCCompiler.c
Linking C executable cmTryCompileExec3662415559
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec3662415559.dir/link.txt --verbose=1
/usr/bin/cc       CMakeFiles/cmTryCompileExec3662415559.dir/testCCompiler.c.o  -o cmTryCompileExec3662415559 -rdynamic 
make[1]: Leaving directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'


Detecting C compiler ABI info compiled with the following output:
Change Dir: /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp

Run Build Command:/usr/bin/make "cmTryCompileExec3363091924/fast"
/usr/bin/make -f CMakeFiles/cmTryCompileExec3363091924.dir/build.make CMakeFiles/cmTryCompileExec3363091924.dir/build
make[1]: Entering directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
/usr/bin/cmake -E cmake_progress_report /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building C object CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o
/usr/bin/cc    -o CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o   -c /usr/share/cmake-2.8/Modules/CMakeCCompilerABI.c
Linking C executable cmTryCompileExec3363091924
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec3363091924.dir/link.txt --verbose=1
/usr/bin/cc     -v CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o  -o cmTryCompileExec3363091924 -rdynamic  
Using built-in specs.
COLLECT_GCC=/usr/bin/cc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/4.8/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 4.8.4-2ubuntu1~14.04.4' --with-bugurl=file:///usr/share/doc/gcc-4.8/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.8 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.8 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --disable-libmudflap --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.8-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.4) 
COMPILER_PATH=/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/:/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/
LIBRARY_PATH=/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/:/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib/:/lib/x86_64-linux-gnu/:/lib/../lib/:/usr/lib/x86_64-linux-gnu/:/usr/lib/../lib/:/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../:/lib/:/usr/lib/
COLLECT_GCC_OPTIONS='-v' '-o' 'cmTryCompileExec3363091924' '-rdynamic' '-mtune=generic' '-march=x86-64'
 /usr/lib/gcc/x86_64-linux-gnu/4.8/collect2 --sysroot=/ --build-id --eh-frame-hdr -m elf_x86_64 --hash-style=gnu --as-needed -export-dynamic -dynamic-linker /lib64/ld-linux-x86-64.so.2 -z relro -o cmTryCompileExec3363091924 /usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crt1.o /usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crti.o /usr/lib/gcc/x86_64-linux-gnu/4.8/crtbegin.o -L/usr/lib/gcc/x86_64-linux-gnu/4.8 -L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu -L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib -L/lib/x86_64-linux-gnu -L/lib/../lib -L/usr/lib/x86_64-linux-gnu -L/usr/lib/../lib -L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../.. CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o -lgcc --as-needed -lgcc_s --no-as-needed -lc -lgcc --as-needed -lgcc_s --no-as-needed /usr/lib/gcc/x86_64-linux-gnu/4.8/crtend.o /usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crtn.o
make[1]: Leaving directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'


Parsed C implicit link information from above output:
  link line regex: [^( *|.*[/\])(ld|([^/\]+-)?ld|collect2)[^/\]*( |$)]
  ignore line: [Change Dir: /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp]
  ignore line: []
  ignore line: [Run Build Command:/usr/bin/make "cmTryCompileExec3363091924/fast"]
  ignore line: [/usr/bin/make -f CMakeFiles/cmTryCompileExec3363091924.dir/build.make CMakeFiles/cmTryCompileExec3363091924.dir/build]
  ignore line: [make[1]: Entering directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp']
  ignore line: [/usr/bin/cmake -E cmake_progress_report /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CMakeFiles 1]
  ignore line: [Building C object CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o]
  ignore line: [/usr/bin/cc    -o CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o   -c /usr/share/cmake-2.8/Modules/CMakeCCompilerABI.c]
  ignore line: [Linking C executable cmTryCompileExec3363091924]
  ignore line: [/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec3363091924.dir/link.txt --verbose=1]
  ignore line: [/usr/bin/cc     -v CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o  -o cmTryCompileExec3363091924 -rdynamic  ]
  ignore line: [Using built-in specs.]
  ignore line: [COLLECT_GCC=/usr/bin/cc]
  ignore line: [COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/4.8/lto-wrapper]
  ignore line: [Target: x86_64-linux-gnu]
  ignore line: [Configured with: ../src/configure -v --with-pkgversion='Ubuntu 4.8.4-2ubuntu1~14.04.4' --with-bugurl=file:///usr/share/doc/gcc-4.8/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.8 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.8 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --disable-libmudflap --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.8-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu]
  ignore line: [Thread model: posix]
  ignore line: [gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.4) ]
  ignore line: [COMPILER_PATH=/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/:/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/]
  ignore line: [LIBRARY_PATH=/usr/lib/gcc/x86_64-linux-gnu/4.8/:/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/:/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib/:/lib/x86_64-linux-gnu/:/lib/../lib/:/usr/lib/x86_64-linux-gnu/:/usr/lib/../lib/:/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../:/lib/:/usr/lib/]
  ignore line: [COLLECT_GCC_OPTIONS='-v' '-o' 'cmTryCompileExec3363091924' '-rdynamic' '-mtune=generic' '-march=x86-64']
  link line: [ /usr/lib/gcc/x86_64-linux-gnu/4.8/collect2 --sysroot=/ --build-id --eh-frame-hdr -m elf_x86_64 --hash-style=gnu --as-needed -export-dynamic -dynamic-linker /lib64/ld-linux-x86-64.so.2 -z relro -o cmTryCompileExec3363091924 /usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crt1.o /usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crti.o /usr/lib/gcc/x86_64-linux-gnu/4.8/crtbegin.o -L/usr/lib/gcc/x86_64-linux-gnu/4.8 -L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu -L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib -L/lib/x86_64-linux-gnu -L/lib/../lib -L/usr/lib/x86_64-linux-gnu -L/usr/lib/../lib -L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../.. CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o -lgcc --as-needed -lgcc_s --no-as-needed -lc -lgcc --as-needed -lgcc_s --no-as-needed /usr/lib/gcc/x86_64-linux-gnu/4.8/crtend.o /usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crtn.o]
    arg [/usr/lib/gcc/x86_64-linux-gnu/4.8/collect2] ==> ignore
    arg [--sysroot=/] ==> ignore
    arg [--build-id] ==> ignore
    arg [--eh-frame-hdr] ==> ignore
    arg [-m] ==> ignore
    arg [elf_x86_64] ==> ignore
    arg [--hash-style=gnu] ==> ignore
    arg [--as-needed] ==> ignore
    arg [-export-dynamic] ==> ignore
    arg [-dynamic-linker] ==> ignore
    arg [/lib64/ld-linux-x86-64.so.2] ==> ignore
    arg [-zrelro] ==> ignore
    arg [-o] ==> ignore
    arg [cmTryCompileExec3363091924] ==> ignore
    arg [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crt1.o] ==> ignore
    arg [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crti.o] ==> ignore
    arg [/usr/lib/gcc/x86_64-linux-gnu/4.8/crtbegin.o] ==> ignore
    arg [-L/usr/lib/gcc/x86_64-linux-gnu/4.8] ==> dir [/usr/lib/gcc/x86_64-linux-gnu/4.8]
    arg [-L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu] ==> dir [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu]
    arg [-L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib] ==> dir [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib]
    arg [-L/lib/x86_64-linux-gnu] ==> dir [/lib/x86_64-linux-gnu]
    arg [-L/lib/../lib] ==> dir [/lib/../lib]
    arg [-L/usr/lib/x86_64-linux-gnu] ==> dir [/usr/lib/x86_64-linux-gnu]
    arg [-L/usr/lib/../lib] ==> dir [/usr/lib/../lib]
    arg [-L/usr/lib/gcc/x86_64-linux-gnu/4.8/../../..] ==> dir [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../..]
    arg [CMakeFiles/cmTryCompileExec3363091924.dir/CMakeCCompilerABI.c.o] ==> ignore
    arg [-lgcc] ==> lib [gcc]
    arg [--as-needed] ==> ignore
    arg [-lgcc_s] ==> lib [gcc_s]
    arg [--no-as-needed] ==> ignore
    arg [-lc] ==> lib [c]
    arg [-lgcc] ==> lib [gcc]
    arg [--as-needed] ==> ignore
    arg [-lgcc_s] ==> lib [gcc_s]
    arg [--no-as-needed] ==> ignore
    arg [/usr/lib/gcc/x86_64-linux-gnu/4.8/crtend.o] ==> ignore
    arg [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crtn.o] ==> ignore
  remove lib [gcc]
  remove lib [gcc_s]
  remove lib [gcc]
  remove lib [gcc_s]
  collapse library dir [/usr/lib/gcc/x86_64-linux-gnu/4.8] ==> [/usr/lib/gcc/x86_64-linux-gnu/4.8]
  collapse library dir [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu] ==> [/usr/lib/x86_64-linux-gnu]
  collapse library dir [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../../../lib] ==> [/usr/lib]
  collapse library dir [/lib/x86_64-linux-gnu] ==> [/lib/x86_64-linux-gnu]
  collapse library dir [/lib/../lib] ==> [/lib]
  collapse library dir [/usr/lib/x86_64-linux-gnu] ==> [/usr/lib/x86_64-linux-gnu]
  collapse library dir [/usr/lib/../lib] ==> [/usr/lib]
  collapse library dir [/usr/lib/gcc/x86_64-linux-gnu/4.8/../../..] ==> [/usr/lib]
  implicit libs: [c]
  implicit dirs: [/usr/lib/gcc/x86_64-linux-gnu/4.8;/usr/lib/x86_64-linux-gnu;/usr/lib;/lib/x86_64-linux-gnu;/lib]
  implicit fwks: []


Determining if files time.h exist passed with the following output:
Change Dir: /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp

Run Build Command:/usr/bin/make "cmTryCompileExec253551286/fast"
/usr/bin/make -f CMakeFiles/cmTryCompileExec253551286.dir/build.make CMakeFiles/cmTryCompileExec253551286.dir/build
make[1]: Entering directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
/usr/bin/cmake -E cmake_progress_report /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building C object CMakeFiles/cmTryCompileExec253551286.dir/CheckIncludeFiles.c.o
/usr/bin/cc  -D_BSD_SOURCE -D_DEFAULT_SOURCE -flto -fno-fat-lto-objects -std=c99 -pedantic -fno-builtin -fno-stack-protector -Wall -Werror=all -Wextra -Werror=extra -Wformat-nonliteral -Werror=format-nonliteral -Winit-self -Werror=init-self -Wconversion -Werror=conversion -Wsign-conversion -Werror=sign-conversion -Wformat-security -Werror=format-security -Wmissing-declarations -Werror=missing-declarations -Wshadow -Werror=shadow -Wstrict-prototypes -Werror=strict-prototypes -Wundef -Werror=undef -Wold-style-definition -Werror=old-style-definition -Wno-stack-protector -Wno-attributes -Werror -Wlogical-op -Werror=logical-op -Wno-error=strict-prototypes -Wno-error=old-style-definition   -o CMakeFiles/cmTryCompileExec253551286.dir/CheckIncludeFiles.c.o   -c /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c
/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c:5:5: warning: function declaration isn’t a prototype [-Wstrict-prototypes]
 int main(){return 0;}
     ^
/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c: In function ‘main’:
/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c:5:5: warning: old-style function definition [-Wold-style-definition]
Linking C executable cmTryCompileExec253551286
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec253551286.dir/link.txt --verbose=1
/usr/bin/cc   -flto -fno-fat-lto-objects -std=c99 -pedantic -fno-builtin -fno-stack-protector -Wall -Werror=all -Wextra -Werror=extra -Wformat-nonliteral -Werror=format-nonliteral -Winit-self -Werror=init-self -Wconversion -Werror=conversion -Wsign-conversion -Werror=sign-conversion -Wformat-security -Werror=format-security -Wmissing-declarations -Werror=missing-declarations -Wshadow -Werror=shadow -Wstrict-prototypes -Werror=strict-prototypes -Wundef -Werror=undef -Wold-style-definition -Werror=old-style-definition -Wno-stack-protector -Wno-attributes -Werror -Wlogical-op -Werror=logical-op -Wno-error=strict-prototypes -Wno-error=old-style-definition    CMakeFiles/cmTryCompileExec253551286.dir/CheckIncludeFiles.c.o  -o cmTryCompileExec253551286 -rdynamic 
make[1]: Leaving directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'


Determining if files unistd.h exist passed with the following output:
Change Dir: /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp

Run Build Command:/usr/bin/make "cmTryCompileExec1346210715/fast"
/usr/bin/make -f CMakeFiles/cmTryCompileExec1346210715.dir/build.make CMakeFiles/cmTryCompileExec1346210715.dir/build
make[1]: Entering directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
/usr/bin/cmake -E cmake_progress_report /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building C object CMakeFiles/cmTryCompileExec1346210715.dir/CheckIncludeFiles.c.o
/usr/bin/cc  -D_BSD_SOURCE -D_DEFAULT_SOURCE -flto -fno-fat-lto-objects -std=c99 -pedantic -fno-builtin -fno-stack-protector -Wall -Werror=all -Wextra -Werror=extra -Wformat-nonliteral -Werror=format-nonliteral -Winit-self -Werror=init-self -Wconversion -Werror=conversion -Wsign-conversion -Werror=sign-conversion -Wformat-security -Werror=format-security -Wmissing-declarations -Werror=missing-declarations -Wshadow -Werror=shadow -Wstrict-prototypes -Werror=strict-prototypes -Wundef -Werror=undef -Wold-style-definition -Werror=old-style-definition -Wno-stack-protector -Wno-attributes -Werror -Wlogical-op -Werror=logical-op -Wno-error=strict-prototypes -Wno-error=old-style-definition   -o CMakeFiles/cmTryCompileExec1346210715.dir/CheckIncludeFiles.c.o   -c /home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c
/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c:5:5: warning: function declaration isn’t a prototype [-Wstrict-prototypes]
 int main(){return 0;}
     ^
/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c: In function ‘main’:
/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp/CheckIncludeFiles.c:5:5: warning: old-style function definition [-Wold-style-definition]
Linking C executable cmTryCompileExec1346210715
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec1346210715.dir/link.txt --verbose=1
/usr/bin/cc   -flto -fno-fat-lto-objects -std=c99 -pedantic -fno-builtin -fno-stack-protector -Wall -Werror=all -Wextra -Werror=extra -Wformat-nonliteral -Werror=format-nonliteral -Winit-self -Werror=init-self -Wconversion -Werror=conversion -Wsign-conversion -Werror=sign-conversion -Wformat-security -Werror=format-security -Wmissing-declarations -Werror=missing-declarations -Wshadow -Werror=shadow -Wstrict-prototypes -Werror=strict-prototypes -Wundef -Werror=undef -Wold-style-definition -Werror=old-style-definition -Wno-stack-protector -Wno-attributes -Werror -Wlogical-op -Werror=logical-op -Wno-error=strict-prototypes -Wno-error=old-style-definition    CMakeFiles/cmTryCompileExec1346210715.dir/CheckIncludeFiles.c.o  -o cmTryCompileExec1346210715 -rdynamic 
make[1]: Leaving directory `/home/nisl/yhy/jerryscript/build/CMakeFiles/CMakeTmp'
```

--- a/jerry-port/default/CMakeLists.txt
+++ b/jerry-port/default/CMakeLists.txt
@@ -54,7 +54,7 @@ INCLUDE (CheckStructHasMember)
 # CHECK_STRUCT_HAS_MEMBER works by trying to compile some C code that accesses the
 # given field of the given struct. However, our default compiler options break this
 # C code, so turn a couple of them off for this.
-set(CMAKE_REQUIRED_FLAGS "-Wno-error=strict-prototypes -Wno-error=old-style-definition")
+set(CMAKE_REQUIRED_FLAGS "-Wno-error=strict-prototypes -Wno-error=old-style-definition -Wno-error=unused-value")
 # tm.tm_gmtoff is non-standard, so glibc doesn't expose it in c99 mode
 # (our default). Define some macros to expose it anyway.
 set(CMAKE_REQUIRED_DEFINITIONS "-D_BSD_SOURCE -D_DEFAULT_SOURCE")

